yk-saito/hotel-booking-dapp
# 7 create connect wallet button

Fix #7 

## What


- Connect to NEAR WALLET
- Disconnect from NEAR WALLET
- Display registered room data in console log

## Test


1. Deploy and start dapp 
```
npm start
```
Expected
```shell
Done deploying to dev-1657821310793-77128084810188
The app is starting! It will automatically open in your browser when ready
Server running at http://localhost:1234
✨ Built in 815ms
```
2. Set room
```shell
near call <dev-1657821310793-77128084810188> set_room '{"id": "1", "name": "second_room", "image": "second.png", "description": "This is Second room", "location": "USA", "price": "1000000000000000000000000"}' --accountId <dev-1657821310793-77128084810188>
```
3. Access dapp & `Sign in`

[http://localhost:1234](http://localhost:1234)
![Untitled](https://user-images.githubusercontent.com/60546319/179395997-777f17f4-42f0-43d6-a725-9d88b2dddf74.jpg)

4. Open console

![pic03](https://user-images.githubusercontent.com/60546319/179396025-a852bae4-ed3f-4342-9ab7-79c83867833a.jpg)

You can show same data
```
near view dev-1657821310793-77128084810188 get_rooms
```

6. You can also `Sign out`
